dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -164,11 +164,11 @@ BEGIN
     LTEXT "", IDC_REASON_DESCRIPTION, 39, 184, 190, 27
     LTEXT "Co&mentariu:", IDC_STATIC, 39, 214, 80, 8
     EDITTEXT IDC_REASON_COMMENT, 39, 224, 190, 35, ES_MULTILINE | WS_VISIBLE | WS_TABSTOP
-    DEFPUSHBUTTON "Con&firmă", IDOK, 150, 276, 55, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", IDCANCEL, 211, 276, 55, 14
+    DEFPUSHBUTTON "&OK", IDOK, 150, 276, 55, 14, WS_GROUP
```

Comment:
This introduces an accelerator collision with the line above
 LTEXT "&Opțiune:", IDC_STATIC, 39, 154, 80, 8

Reply:
I do confirm that The collisions are gone now after switching to O&K everywhere. I do still insist on a screenshot. Download a Romanian 2k3 if you don't have it on your current more modern Windows.